src/modules/task-show.js

```diff
@@ -0,0 +1,130 @@
+/* eslint-disable import/no-cycle */
+import { deleteTask, storageUpdate } from './localstorage.js';
+import { inputNewTasks, buttonsDeactive } from '../index.js';
+
+export const toDoDivTask = document.getElementById('sect__one__cont__items__item__two');
+const deletToDoTasks = document.querySelector('.item__three__btn__del');
+let clearToDoTasks;
+let modifyToDoTasks;
+let counter;
+let toDotasksupdate = '';
+
+/* || FUNCTION TO DISPLAY THE TASKS */
+export const showToDoTasks = () => {
+  if (Object.keys(localStorage).length > 0) {
+    toDoDivTask.style.display = 'inline-block';
+  } else {
+    toDoDivTask.style.display = 'none';
+  }
+  /* || Clear the tasks */
+  toDoDivTask.innerHTML = '';
+  /* || Fetch All The Keys in local storage */
+  let toDoTasks = Object.keys(localStorage);
+  toDoTasks = toDoTasks.sort();
+
+  toDoTasks.forEach((key) => {
+    /* || Get all values */
+    const toDoValue = localStorage.getItem(key);
+
+    const toDoInnerDiv = document.createElement('p');
+    toDoInnerDiv.classList.add('p__cont__task');
+    toDoInnerDiv.setAttribute('id', key);
+    toDoInnerDiv.innerHTML = `<p id="todo__name">${key.split('_')[1]}</p>`;
+    /* || localstorage would store boolean as string so we parse it to boolean back */
+    const buttonModify = document.createElement('button');
+    buttonModify.classList.add('edit__btn');
```

Comment:
- [ ] [OPTIONAL] Even if this is not among the project requirements in this current project It should be noted that the task should be saved in local storage as an object with properties including the `completed` that accepts a boolean value and also all tasks should be in an array. Currently, they save as separate
👇 👇 
![image](https://user-images.githubusercontent.com/90222110/202859286-965e64fe-b8ee-4d1d-9a3e-7f5b08c8fddd.png)

Please head back to the previous project requirements and see how you were supposed to implement it